### PR TITLE
Proposal: rename game.size to game.gameSize as well

### PR DESCRIPTION
**Pull request by luanpotter**
Proposal: Since we recently changed component.size to gameSize, though game.size is more clear this makes it more consistent. Not 100% convinced though, so thoughts welcome.

### CODE REVIEW

**Review by erickzanardo — APPROVED** [review]
You just forget the changelog, other than that, LGTM

File: lib/game/base_game.dart [hunk 1]
```diff
@@ -31,7 +31,7 @@ class BaseGame extends Game with FPSCounter {
   final List<Component> _removeLater = [];
 
   /// Current game viewport size, updated every resize via the [resize] method hook
-  Vector2 size;
+  Vector2 gameSize;
```
Comment (spydon) [inline lib/game/base_game.dart:34]: I think it makes sense on the component that it has gameSize named gameSize, but on the game which is another type of object I don't think we have to follow that convention, that the size field on game is the game size should be clear and just adds redundancy to call game.gameSize imho.

Comment (erickzanardo) [reply lib/game/base_game.dart:34]: On a second thought, I think I agree with @spydon !

**Review by renancaraujo — APPROVED** [review]

File: lib/game/game.dart [hunk 2]
```diff
@@ -40,7 +40,7 @@ abstract class Game {
   /// This is the resize hook; every time the game widget is resized, this hook is called.
   ///
   /// The default implementation does nothing; override to use the hook.
-  void resize(Vector2 size) {}
+  void onGameResize(Vector2 gameSize) {}
```
Comment (erickzanardo) [inline lib/game/game.dart:43]: This could be just `onResize` what do you think?

**Review by erickzanardo — COMMENTED** [review]
I think we should leave the property named size, but the `resize` method, could be improved, my suggestion is just `onResize`.

**Comment by luanpotter** [comment]
@erickzanardo sgtm. I will close this and make a followup PR just with the method name change.

I will use this PR as an example on a small talk about contributing to open source next week, so please leave it this to me :)

**Pull request closed without merging**